addons/stock/models/stock_picking.py

```diff
@@ -489,16 +489,17 @@ def _compute_state(self):
             })
             picking_move_lines[picking_id.id].add(move.id)
         for picking in self:
-            if not picking_moves_state_map[picking.id]:
+            picking_id = (picking.ids and picking.ids[0]) or picking.id
```

Comment:
picking.ids[:1] is not enough?

Reply:
`[:1]` returns a list so what do you mean with `enough` ?

Reply:
Oups my bad